<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SH3 Dynamic Campaign - Installer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;SH3 Dynamic Campaign&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;DISCLAIMER: &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;1. I hereby understand, nor will I disobey, that &amp;quot;SH3 Dynamic Campaign&amp;quot;, or any of its derivatives or modified versions, is/are not to be used for commercial or monetary means.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;2. I understand that &amp;quot;SH3 Dynamic Campaign&amp;quot; is to be distributed freely through any means necessary. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;3. I understand that &amp;quot;SH3 Dynamic Campaign&amp;quot; CAN be improved upon or modified without the permission of its creator or contributors. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;4. I also understand that the creator of &amp;quot;SH3 Dynamic Campaign&amp;quot; does not take credit for the work of other contributors even if permission to use their work was never granted. The creator thanks them for their work, and without it &amp;quot;SH3 Dynamic Campaign&amp;quot; would have never been possible.  &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>571</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Introduction</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>441</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(://images/3a.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Next">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Prev">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SH3Saves">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>508</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Location where Silent Hunter III stores save games, patrols and career files. The directory is often named &amp;quot;SH3&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_SH3Saves">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>521</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Location where Silent Hunter III stores save games, patrols and career files. The directory is often named &amp;quot;SH3&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Where is the location of Silent Hunter 3 save game files?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>391</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Check if playing with SH3 Commander</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SH3Commander">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>521</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Directory where SH3 Commander is installed. This directory is often named &amp;quot;SH3 Commander&amp;quot; and contains &amp;quot;SH3Cmdr.exe&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Where is SH3 Commander installed?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SH3Commander">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>508</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Directory where SH3 Commander is installed. This directory is often named &amp;quot;SH3 Commander&amp;quot; and contains &amp;quot;SH3Cmdr.exe&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>139</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_Agree">
       <property name="text">
        <string>I Agree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Disagree">
       <property name="text">
        <string>I Disagree</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>510</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_Para">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>This setup process will guide you through the installation of SH3 Dynamic Campaign mod 
for Silent Hunter 3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SH3Install">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>508</width>
      <height>57</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Directory where Silent Hunter III is installed. Directory is often named &amp;quot;SilentHunterIII&amp;quot; and contains &amp;quot;sh3.exe&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Where is Silent Hunter 3 installed?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SH3Install">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>508</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Directory where Silent Hunter III is installed. Directory is often named &amp;quot;SilentHunterIII&amp;quot; and contains &amp;quot;sh3.exe&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_installItems">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 14pt &quot;MS Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Items to install:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>440</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_ProgressBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:normal;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Shortcut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>541</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Create SH3 Dynamic Campaign Start Menu Shortcut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DirSaves">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>276</y>
      <width>25</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DirSH3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>227</y>
      <width>25</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DirSH3Cmdr">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>365</y>
      <width>25</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
